samples/unit-tests/axis/ordinal/demo.js

```diff
@@ -564,7 +568,10 @@ QUnit.test('Ordianl axis, data grouping and boost module, #14055.', assert => {
     });
     assert.ok(
         chart.xAxis[0].ordinal.positions.length &&
-            chart.series[0].boostClipRect,
+        (
+            chart.series[0].boostClipRect ||
+            chart.boostClipRect
+        ),
```

Comment:
These changes (and the ones above) are related to the fact that now this chart has two supported series types, so a common `chart.boostClipRect` was created instead of a single `series[x].boostClipRect`.